### PR TITLE
Fix wrong i18n:translate tag

### LINKED ISSUE #69: ${DYNAMIC_CONTENT} is shown in po files when translating this package
See https://github.com/plone/plone.app.linkintegrity/commit/d7797fcd36f4546ca74019e56fbf557e31e551ae#commitcomment-31366546 for context

### PR DESCRIPTION
**Pull request by erral**
This fixes #69

### CODE REVIEW

**Comment by erral** [comment]
@jenkins-plone-org please run jobs

**Comment by erral** [comment]
@jenkins-plone-org please run jobs

File: plone/app/linkintegrity/browser/delete_confirmation_info.pt [hunk 1]
```diff
@@ -65,10 +65,8 @@
             </span><br/>
             <ul>
               <li tal:define="objects view/objects"
-                  tal:repeat="item python:range(3) "
-                  i18n:translate="">
-                <span tal:replace="python: breach_count[content][item]" />
-                <span i18n:translate="" tal:replace="python: objects[item]" />
+                  tal:repeat="item python:range(3) ">
+                <span tal:replace="python: breach_count[content][item]" /> <span i18n:translate="" tal:replace="python: objects[item]" />
```
Comment (jensens) [inline plone/app/linkintegrity/browser/delete_confirmation_info.pt]: I think even here the `i18n:translate` results in dynamic content. There is nothing to be translated by Gettext, so no `i18n:translate` is needed at all.

Comment (erral) [reply plone/app/linkintegrity/browser/delete_confirmation_info.pt]: Objects is the list of the possible breach names: https://github.com/plone/plone.app.linkintegrity/blob/master/plone/app/linkintegrity/browser/info.py#L195 and item is the integer index to access one of those values, so the ' i18n:translate' forces the translation of those possible breach names.

Comment (jensens) [reply plone/app/linkintegrity/browser/delete_confirmation_info.pt]: Sorry, but if the value of `breach_count[content][item]` is already a ` zope.i18nmessageid.message.Message` the template engine translates it anyway on output, there is no additional `i18n:translate` needed. And according to the given code reference those are created by a message factory.

Comment (erral) [reply plone/app/linkintegrity/browser/delete_confirmation_info.pt]: Then I will just remove the i18n and we are done.

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Comment by erral** [comment]
@jenkins-plone-org please run jobs

**Pull request merged**